packages/ai-workspace-agent/src/common/coder-replace-prompt-template.ts

```diff
@@ -46,8 +46,8 @@ ${withSearchAndReplace ? '**~{changeSet_replaceContentInFile}**\n' : ''}
 ### Guidelines for Proposing Code Changes
 - For each proposed change:
   - **Retrieve Current Content**: Use getFileContent to get the latest content of the target file.
-  - **Change Content**: Use ${withSearchAndReplace ? 'changeSet_writeChangeToFile or changeSet_replaceContentInFile' : 'changeSet_writeToFileChange'}\n
-   to suggest file changes to the user.`,
-    ...(withSearchAndReplace ? { variantOf: CODER_DEFAULT_PROMPT_TEMPLATE_ID } : {}),
+  - **Change Content**: Use ${withSearchAndReplace ? 'changeSet_writeChangeToFile or changeSet_replaceContentInFile' : 'changeSet_writeToFileChange'} 
```

Comment:
These ids are not correct, are they? I don't think there is a `changeSet_writeToFileChange`.
```suggestion
  - **Change Content**: Use ${withSearchAndReplace ? 'changeSet_writeChangeToFile or changeSet_replaceContentInFile' : 'changeSet_writeChangeToFile'} 
```

Reply:
I think we should merge the `## Propose Code Changes` and `### Guidelines for Proposing Code Changes` to a single section. Seems rather redundant. And we could also then only mention each function once so we avoid accidental errors like these.